docs/source/usage/plugins/radiation.rst

```diff
@@ -142,38 +142,30 @@ By using ``NyquistFactor = 0.5`` for periodic boundary conditions, particles tha
 Form factor
 """""""""""
 
-The *form factor* is still an experimental method trying to consider the shape of the macro particles when computing the radiation.
-By default, it should be switched off by setting ``__COHERENTINCOHERENTWEIGHTING__`` to zero. 
-
-.. code:: cpp
-
-   // correct treatment of coherent and incoherent radiation from macroparticles
-   // 1 = on (slower and more memory, but correct quantitative treatment)
-   // 0 = off (faster but macroparticles are treated as highly charged, point-like particle)
-   #define __COHERENTINCOHERENTWEIGHTING__ 0
-
+The *form factor* is a method, which considers the shape of the macro particles when computing the radiation.
 
 If switched on, one can select between different macro particle shapes. 
```

Comment:
Currently, only selecting a shape is possible. The "off" option is equavalent to `radFormFactor_coherent` since #1937.

Reply:
```suggestion
One can select between different macro particle shapes. 
```